localstack/services/awslambda/provider.py

```diff
@@ -3128,14 +3147,12 @@ def delete_layer_version(
         if version_number < 1:
             raise InvalidParameterValueException("Layer Version Cannot be less than 1", Type="User")
 
-        # TODO: test cross-region? (e.g. with functions it doesnt work and raises an exception)
```

Comment:
This does not seem tested now, are we sure this works? Otherwise we might not want to remove that TODO

Reply:
Yes, the point of that comment was however to snapshot test if it is possible in aws or not, and apply the restrictions.